test/Microsoft.ML.TestFrameworkCommon/Microsoft.ML.TestFrameworkCommon.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!--<PackageReference Include="Microsoft.ML" Version="1.3.1" />
+    <PackageReference Include="Microsoft.ML.Ensemble" Version="0.15.1" />
+    <PackageReference Include="Microsoft.ML.FastTree" Version="1.3.1" />-->
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
```

Comment:
I am guessing this is no longer needed? If not, can you please delete it?

Reply:
Yes, will remove these

---
In reply to: [337122073](https://github.com/dotnet/machinelearning/pull/4346#discussion_r337122073) [](ancestors = 337122073)